src/components/NewNursery/index.tsx

```diff
@@ -14,13 +14,19 @@ import useSnackbar from 'src/utils/useSnackbar';
 import { getAllNurseries } from 'src/utils/organization';
 import TfMain from 'src/components/common/TfMain';
 import { useOrganization } from 'src/providers/hooks';
+import TimeZoneSelector from '../TimeZoneSelector';
+import { TimeZoneDescription } from 'src/types/TimeZones';
+import { Checkbox } from '@terraware/web-components';
+import { useDefaultTimeZone } from 'src/utils/useTimeZoneUtils';
 
 export default function NurseryView(): JSX.Element {
   const { selectedOrganization, reloadData } = useOrganization();
   const [nameError, setNameError] = useState('');
   const [descriptionError, setDescriptionError] = useState('');
   const snackbar = useSnackbar();
   const theme = useTheme();
+  const [orgTZChecked, setOrgTZChecked] = useState<boolean>(false);
+  const defaultTimeZone = useDefaultTimeZone();
```

Comment:
i messed up, i need to update the utility to consider if editing or read-only.
For read-only we won't use the user's time zone. For edit we will.
Can you add an optional 'forEdit?: boolean' in the utility.
If forEdit, then also use user's time zone, otherwise don't use the user's time zone.

Reply:
actually let me send a PR out for this, since it's my mess up

Reply:
I have this out https://github.com/terraware/terraware-web/pull/1018 also added a `useFacilityTimeZone` which handles the fallback - so client code finds it easier to use

Reply:
I don't get it. If editing, we default the user's time zone, but if it is read-only and it doesn't have any timezone value, what are we going to show?

Reply:
Looking at your PR, it seems for ready-only we show UTC time zone, and not the user timezone. Why is that?

Reply:
1) in display mode: the nursery time zone shows what the system/BE uses. nusery-timezone || org-timezone || UTC.
2) in edit mode, we are trying to suggest a time zone to use. If org doesn't have a time zone set, we are suggesting user to use his/her time zone value (we will almost never end up suggesting UTC fallback since user time zone is set by us).
Basically - in edit mode, we are 'suggesting' a value to use.
In read-only mode, we are showing what is actually being used.
Not sure if this clarifies?

Reply:
Actually, this is still technically valid - since setting the org or user time zone may fail. And data model has time zone as optional. I can take care of this later if you wish.